.github/workflows/build.yml

```diff
@@ -37,7 +39,7 @@ jobs:
     steps:
     # Pinned 1.0.0 version
     - uses: marocchino/action-workflow_run-status@54b6e87d6cb552fc5f36dbe9a722a6048725917a
-      if: github.event_name != 'workflow_dispatch'
+      if: github.event_name == 'workflow_run'
     - uses: actions/checkout@v3
       with:
         ref: ${{ github.event.workflow_run.head_sha }}
```

Comment:
We also need to fix this.

Reply:
So i believe when this is empty, it will just reference the default develop branch. I guess if I want to run it against release branch, I should do the similar matrix thing in cdap-build?

Reply:
Scheduled builds only run on `default` branch. If you want to run it against release branch also on schedule that you might need to build something similar hack we do in `cdap-build`.

If you only need to run it against release branch using `workflow_dispatch` that can be done by adding a step which computes the branch based on github event.